cubic-server/default/DefaultWorld.cpp

```diff
@@ -6,14 +6,21 @@ DefaultWorld::DefaultWorld(WorldGroup *worldGroup)
     this->_dimensions.emplace("end", std::make_shared<TheEnd>(reinterpret_cast<World *>(this)));
     this->_dimensions.emplace("nether", std::make_shared<TheNether>(reinterpret_cast<World *>(this)));
     this->_dimensions.emplace("overworld", std::make_shared<Overworld>(reinterpret_cast<World *>(this)));
+    for (auto [_, dim] : this->_dimensions)
+        _processingThreads.push_back(new std::thread([dim](){
+            while (1) {
+                dim->dimensionLock.acquire();
+                dim->tick();
```

Comment:
This will not run at 20t/s

Reply:
Btw, do you need to release the lock ? I am not familiar with this type

Reply:
It is running at 20 ticks per second, the semaphore is released by the global tick rate. Therefore it will not run faster than 20 ticks per seconds, except if the computation is lagging behind, which would make it not wait for the 50ms to start computing

Reply:
big brain move